control-plane/agents/core/src/server.rs

```diff
@@ -75,6 +76,11 @@ pub(crate) struct CliArgs {
     /// Trace rest requests to the Jaeger endpoint agent
     #[structopt(long, short)]
     jaeger: Option<String>,
+    /// The GRPC Server URLs to connect to
+    /// (supports the http/https schema)
+    /// Default: 0.0.0.0:50051
+    #[structopt(long, short, default_value = "https://0.0.0.0:50051")]
```

Comment:
We can put this default value in utils rather than a string here, so it can be reused.